### PR TITLE
feat: installed and configured Simple JWT

### PR DESCRIPTION
**Pull request by casneno**

### CODE REVIEW

File: secretsanta/urls.py [hunk 1]
```diff
@@ -15,8 +15,13 @@
...
-from django.urls import path
+from django.urls import path, include
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView, TokenVerifyView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api-auth/', include('rest_framework.urls')), #browsable API
+    path('api/token', TokenObtainPairView.as_view(), name='token_obtain_pair'), #Simple JWT
+    path('api/token/refresh', TokenRefreshView.as_view(), name='token_refresh'),#Simple JWT
+    path('api/token/verify/', TokenVerifyView.as_view(), name='token_verify'), # Simple JWT, allow API users to verify HMAC-signed tokens without having access to your signing key
```
Comment (silvinovieira) [inline secretsanta/urls.py]: I think we won't use this endpoint for now.

Comment (casneno) [reply secretsanta/urls.py]: Agreed. I will remove it

File: secretsanta/urls.py [hunk 2]
```diff
@@ -15,8 +15,13 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView, TokenVerifyView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api-auth/', include('rest_framework.urls')), #browsable API
```
Comment (silvinovieira) [inline secretsanta/urls.py]: Remove comment.

Comment (silvinovieira) [reply secretsanta/urls.py]: @casneno

**Review by silvinovieira — APPROVED** [review]

**Pull request merged**